apps/web-giddh/src/app/services/settings.integration.service.ts

```diff
@@ -625,4 +625,42 @@ export class SettingsIntegrationService {
             return data;
         }), catchError((e) => this.errorHandler.HandleCatch<any, any>(e)));
     }
+
+    /**
+     * Multi-purpose CRUD method for PayU integration.
+     * @param method HTTP method type ('GET' | 'POST' | 'PUT' | 'DELETE')
+     * @param payload Optional payload for POST/PUT/DELETE
+     * @returns Observable<BaseResponse<any, any>>
+     * @memberof SettingsIntegrationService
+     */
+    public payuCrudOperation(
+        method: 'GET' | 'POST' | 'PUT' | 'DELETE',
```

Comment:
```suggestion
        method: ApiMethodType,
```
export type ApiMethodType = 'get' | 'post' | 'put' | 'patch' | 'delete';